polyfill/ci_test.sh

```diff
@@ -17,39 +13,52 @@ else
   cd ..
 fi
 
-PRELUDE=script.js
-if [ "x$COVERAGE" = xyes ]; then
-    nyc instrument script.js > script-instrumented.js
-    PRELUDE=script-instrumented.js
-fi
-
-cd test/
-
 if [ "$(uname)" = 'Darwin' ]; then
   threads=$(sysctl -n hw.logicalcpu)
 else
   threads=$(nproc --ignore 1)
 fi
 if [ $threads -gt 2 ]; then threads=2; fi
 
-test262-harness \
-  -t $threads \
-  -r json \
-  --reporter-keys file,rawResult,result,scenario \
-  --test262Dir ../test262 \
-  --prelude "../$PRELUDE" \
-  --transformer ./transform.test262.js \
-  "$TESTS" \
-  | ./parseResults.py
-RESULT=$?
+if [ "x$COVERAGE" = xyes ]; then
+  nyc instrument script.js > script-instrumented.js
+  PRELUDE=script-instrumented.js
 
-cd ..
+  cd test/
 
-if [ "x$COVERAGE" = xyes ]; then
-    nyc report -t coverage/tmp/transformer --reporter=text-lcov > coverage/test262.lcov
-fi
+  test262-harness \
```

Comment:
Repeating this invocation increases the chance of them going out of sync; using `TRANSFORMER_ARG=--transformer ./transform.test262.js` or something like that should work.

Reply:
good idea. done.